build.sbt

```diff
@@ -302,12 +318,12 @@ def minorVersion(version: String): String = {
 val latestVersion = settingKey[String]("Latest stable released version")
 latestVersion in ThisBuild := {
   val snapshot = (isSnapshot in ThisBuild).value
-  val stable = (isVersionStable in ThisBuild).value
+  val stable   = (isVersionStable in ThisBuild).value
 
   if (!snapshot && stable) {
     (version in ThisBuild).value
   } else {
-    (previousStableVersion in ThisBuild).value.get
+    (previousStableVersion in ThisBuild).value.getOrElse("UNKNOWN")
```

Comment:
I've hit an error on my GH actions with this line. I'm not sure the `getOrElse` workaround is appropriate

Reply:
I don't know either, let's merge and we'll see.